packages/synthetics/data_stream/browser/fields/summary.yml

```
@@ -11,24 +11,27 @@
       type: integer
       description: >
         The number of endpoints that failed
+
     - name: status
       type: keyword
       description: >
         The status of this check as a whole. Either up or down.
+
     - name: attempt
       type: short
       description: >
         When performing a check this number is 1 for the first check, and increments in the event of a retry.
+
     - name: max_attempts
       type: short
       description: >
         The maximum number of checks that may be performed. Note, the actual number may be smaller.
+
     - name: final_attempt
       type: boolean
       description: >
         True if no further checks will be performed in this retry group.
+
     - name: retry_group
       type: keyword
-      description: >
-        A unique token used to group checks across attempts.        
-
+      description: "A unique token used to group checks across attempts.        \n"
```

There's lots of white space in the end, is this intentional?
```suggestion
      description: "A unique token used to group checks across attempts.\n"
```

---

++ it does seem strange. I'm a little disinclined to change it without proper investigation into its usage in Kibana, and that would be a bit out of scope for this patch. We want it to be as transparent as possible. I did ask some folks and they were unsure why we did this, which makes me _not_ want to change it at this time, if that makes sense.